### ISSUE TITLE
[Dropdown] `.selection`’s menu shouldn’t be scrolled down when its first element is a `.header`

### ISSUE THREAD

**Issue by batata004** [Semantic-Org/Semantic-UI#5099]
Hi,

When I have a dropdown element and I just focus it, it does not display the first element, it is displaying the second element, really really weird. I have a header element and it should be shown before any other. See here:

https://jsfiddle.net/88kh769y/2/

Just click inside the dropdown (and dont type anything). You will see it will not display IMÓVEIS (which should be the first item). Scroll up in the list and you will see IMÓVEIS is the first but it's not shown at first.

Is there a workaround I can implement while this bug is not fixed? I am using this component in a final website and a user reported it to me and it's not good for my business.

**Comment by kmd1970** [comment]
Workaround.

```javascript
$(".elemento_categoria").dropdown({
     onShow: function (){
      $(this).find('.menu').animate({ scrollTop: 0 }, "fast");
   }               
});
```

**Comment by batata004** [comment]
THANKS MAN, really nice workaround :) Is it possible that someone can mark this as a bug and fix it so in the next versions this workaround will not be needed?

**Comment by kmd1970** [comment]
> THANKS MAN, really nice workaround :) Is it possible that someone can mark this as a bug and fix it so in the next versions this workaround will not be needed?

The admin/mods will determine that after they evaluate this issue.

**Comment by Banandrew** [comment]
Hi @batata004, I don’t think headers ever worked with `.selection` dropdowns or dropdowns that generally mimic the look and the functionality of default form elements. It does make sense to enable headers for such elements for consistency.

**Comment by batata004** [comment]
@Banandrew thanks but semantic ui docs ifself present headers. Check this -> http://semantic-ui.com/modules/dropdown.html

Go to the section SEARCH IN-MENU. You will see in the code there are elements with header class AND search input to filter data. If your docs say I can use HEADER with seach element, and this is causing problems, I assume there is a bug not an "enhancement", right?

**Comment by Banandrew** [comment]
@batata004 no, because it’s a `.floating` (or `.pointing`, it doesn’t matter) dropdown’s menu, and not `.selection`; the input inside of it doesn’t change the fact that it’s a `.floating` dropdown.

**Comment by batata004** [comment]
@Banandrew you are very correct, I removd selection class and it worked perfectly despite the fact that the menu window is very ugly and using "selection" makes it much much more beautiful. Is there any chance the community fix this to selection? As I see, there is no point leaving "selection" out of this solution, it should work the same as floating, pointing cause the use case is very similar, if not equal.

**Comment by Banandrew** [comment]
@batata004 Can’t speak for the community, but I have a lot of smaller issues in my backlog that I wanted to address first. I can take a look at it either next week, or the week after the next one.

**Comment by batata004** [comment]
@Banandrew sure! :) I also wouldnt like you to address less important stuff and leave relevant bugs unsolved! Follow your priority, I can wait. The workaround is perfect so no reason to rush!

**Comment by Banandrew** [comment]
My apologies, because I completely missed the point of the issue; the headers do work with `.selection` dropdowns, it’s just that when the first `.menu` element is a `.header`, the menu scrolls itself down until it finds an `.item`.

#1727 is slightly related.

**Comment by icepeng** [comment]
@Banandrew I found it to be a problem. I solved it by adding a hidden item to the first of the menu. That item can also work as placeholder.
Code: https://github.com/Semantic-Org/Semantic-UI/issues/1727#issuecomment-286959152

**Comment by batata004** [comment]
@icepeng  indeed your solutino works too! So far I am using @kmd1970 solution which uses JS. Your solution is a little bit easier but it would be nice to have this bug solved :)

**Comment by lubber-de** [comment]
Fixed in Fomantic-UI by https://github.com/fomantic/Fomantic-UI/pull/957
See your adjusted jsfiddle here: https://jsfiddle.net/pftax9Ly/
